frinx-uniconfig/user-guide/network-management-protocols/uniconfig_gnmi/readme.md

```diff
@@ -0,0 +1,208 @@
+---
+label: UniConfig gNMI
+order: 8000
+---
+
+# UniConfig gNMI
+
+## Introduction
+
+The gNMI (gRPC Network Management Interface) southbound plugin enables Frinx 
+UniConfig to communicate with a device through gNMI. It provides the mechanism to
+install, manipulate, and delete the configuration of network devices, and also to view operational data.
+
+The gNMI southbound plugin follows a fully model-driven approach, similar 
+to CLI or NETCONF southbound plugins. However, the difference is that it uses protocol buffer (proto3) 
+for service structure modeling within it's RPCs instead of YANG.
```

Comment:
the first question of reader could be - why do we need YANG models during installation of the device if data model is written using proto schemas?

Reply:
ack